README.md

```diff
@@ -61,159 +22,82 @@ In one terminal window, run:
+2. `yarn run start-console` (requires [Docker](https://www.docker.com) or [Podman 3.2.0+](https://podman.io))
 
-This will run the OpenShift console in a container connected to the cluster
-you've logged into. The plugin HTTP server runs on port 9001 with CORS enabled.
-Navigate to <http://localhost:9000/example> to see the running plugin.
+This will run the OpenShift console in a container connected to the cluster you've logged into. The plugin HTTP server runs on port 9001 with CORS enabled. Navigate to <http://localhost:9000/k8s/ns/default/nfspvc.dana.io~v1alpha1~NfsPvc> to see the running plugin.
 
-#### Running start-console with Apple silicon and podman
+#### Option 2: Docker + VSCode Remote Container
 
-If you are using podman on a Mac with Apple silicon, `yarn run start-console`
-might fail since it runs an amd64 image. You can workaround the problem with
-[qemu-user-static](https://github.com/multiarch/qemu-user-static) by running
-these commands:
+You can use Docker Compose with VSCode Remote Containers for easy development. Set up the environment variables in a `.dev.env` file:
 
 ```bash
-podman machine ssh
-sudo -i
-rpm-ostree install qemu-user-static
-systemctl reboot
-```
-
-### Option 2: Docker + VSCode Remote Container
-
-Make sure the
-[Remote Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
-extension is installed. This method uses Docker Compose where one container is
-the OpenShift console and the second container is the plugin. It requires that
-you have access to an existing OpenShift cluster. After the initial build, the
-cached containers will help you start developing in seconds.
-
-1. Create a `dev.env` file inside the `.devcontainer` folder with the correct values for your cluster:
-
-```bash
-OC_PLUGIN_NAME=console-plugin-template
+OC_PLUGIN_NAME=nfs-pvc-plugin
 OC_URL=https://api.example.com:6443
 OC_USER=kubeadmin
 OC_PASS=<password>
 ```
 
-2. `(Ctrl+Shift+P) => Remote Containers: Open Folder in Container...`
-3. `yarn run start`
-4. Navigate to <http://localhost:9000/example>
+Open the folder in a remote container (`Ctrl+Shift+P` -> `Remote Containers: Open Folder in Container...`) and run:
 
-## Docker image
+```bash
+yarn run start
+```
 
-Before you can deploy your plugin on a cluster, you must build an image and
-push it to an image registry.
+Then navigate to <http://localhost:9000/k8s/ns/default/nfspvc.dana.io~v1alpha1~NfsPvc>.
 
-1. Build the image:
+## Plugin Deployment
 
-   ```sh
-   docker build -t quay.io/my-repository/my-plugin:latest .
-   ```
+### Build the Plugin Image
+
+To deploy the plugin to an OpenShift cluster, build and push the Docker image to an image registry.
 
-2. Run the image:
+1. Build the image:
 
    ```sh
-   docker run -it --rm -d -p 9001:80 quay.io/my-repository/my-plugin:latest
+   docker build -t quay.io/my-repository/nfs-pvc-plugin:latest .
```

Comment:
Add CI to this repository which builds the image on releases and change the README.md here to use `ghcr.io` instead of `quay.io`. For example: https://github.com/dana-team/cert-external-issuer/blob/main/.github/workflows/release.yaml